nservicebus/hosting/azure-functions-service-bus/index.md

```diff
@@ -20,6 +20,8 @@ snippet: asb-function-isolated-configuration
 
 The Azure Service Bus trigger is automatically generated by the `NServiceBusTriggerFunction` attribute which will forward incoming messages to the appropriate message handlers.
 
+The Azure Service Bus trigger can be manually declared instead of relying on the auto-generated trigger. See the article on [custom Azure Functions triggers](/nservicebus/hosting/azure-functions-service-bus/custom-triggers.md) for more information.
```

Comment:
```suggestion
Should customization of the Azure Service Bus trigger be required, it can be manually declared instead of relying on the auto-generated trigger. See the article on [custom Azure Functions triggers](/nservicebus/hosting/azure-functions-service-bus/custom-triggers.md) for more information.
```